### DISCUSSION TITLE
New function "Freeze Bandwidth"

### DISCUSSION THREAD

**Discussion by andimik** [PE5PVB/TEF6686_ESP32#91] [Ideas]
It's quite annoying that a click on the **BW button** will change (for instance) from `AUTO BW` back to the **first entry at 56 kHz**, then we need 18 (?) clicks to get back to `AUTO BW`. 

So I suggest a function freezing the BW to the current used one or even to `AUTO BW`

**Comment by PE5PVB** [comment]
You can also just not press it ;-)
